samples/videoplayer/README.md

```diff
@@ -7,7 +7,7 @@ ffmpeg and SDL2 is needed for that to work, i.e.
      port install ffmpeg-devel
      brew install ffmpeg sdl2
      apt install libavcodec-dev libavformat-dev libavutil-dev libswscale-dev \
-                 libavresample-dev
+                 libswresample-dev
```

Comment:
Thanks for this change, however it would be awesome, if you could detect in Gradle which ffmpeg is installed and support both old and new one with klib, for example by adding conditionally compiled wrapper functions into the .def file.

Reply:
Is it really needed? libswresample is copyrighted "2011-2014", so it's here at least five years.

Reply:
OK, let's keep it as is.